specification/columns/resourceid.md

```diff
@@ -2,7 +2,7 @@
 
 A Resource ID is an identifier assigned to a [*resource*](#glossary:resource) by the provider. The Resource ID is commonly used for cost reporting, analysis, and allocation scenarios.
 
-The ResourceId column MUST be present in a FOCUS dataset when the provider supports billing based on provisioned resources. This column MUST be of type String. The ResourceId value MAY be a nullable column as some cost data [*rows*](#glossary:row) may not be associated with a *resource*. ResourceId MUST appear in the cost data if an identifier is assigned to a *resource* by the provider. ResourceId SHOULD be a fully-qualified identifier that ensures global uniqueness within the provider.
+The ResourceId column MUST be present in a FOCUS dataset when the provider supports billing based on provisioned resources. This column MUST be of type String. The ResourceId value MAY be a nullable column as some cost data [*rows*](#glossary:row) may not be associated with a *resource*. ResourceId MUST appear in the cost data if an identifier is assigned to a *resource* by the provider and MUST be null when [CommitmentDiscountStatus](#commitmentdiscountstatus) is "Unused". ResourceId SHOULD be a fully-qualified identifier that ensures global uniqueness within the provider.
```

Comment:
Why are we saying this. The resource that incurs the charge when a reservation is not used is the reservation.

Reply:
I see your point and could go either way here.  On one hand, the id is present in `CommitmentDiscountId`, and on the other, duplicating does avoid a `null` value.

Reply:
This is my concern with the PR in that it makes the identification of the applicable resource that incurred the charge inconsistent. To answer the what incurred the charge I would have to query conditionally rather than just use the resourceId. 

```suggestion
The ResourceId column MUST be present in a FOCUS dataset when the provider supports billing based on provisioned resources. This column MUST be of type String. The ResourceId value MAY be a nullable column as some cost data [*rows*](#glossary:row) may not be associated with a *resource*. ResourceId MUST appear in the cost data if an identifier is assigned to a *resource* by the provider. In the event that [CommitmentDiscountStatus](#commitmentdiscountstatus) is "Unused", ResourceId should be the identifier of the unused commitment. ResourceId SHOULD be a fully-qualified identifier that ensures global uniqueness within the provider.
```

Reply:
5-0 poll to remove this change, removed now.